packages/fxa-content-server/app/scripts/lib/channels/senders/web-channel.js

```diff
@@ -122,8 +122,9 @@ function formatEventDetail(win, eventDetail) {
   // See https://bugzilla.mozilla.org/show_bug.cgi?id=1275616 and
   // https://bugzilla.mozilla.org/show_bug.cgi?id=1238128
   if (
-    (userAgent.isFirefoxDesktop() || userAgent.isFirefoxAndroid()) &&
-    userAgent.browser.version >= 50
+    userAgent.isFirefoxDesktop() ||
+    userAgent.isFirefoxAndroid() ||
+    userAgent.isThunderbirdDesktop()
```

Comment:
Just noting, this change affects Backbone web channel messages, but not React. Our email-first/index page and Sync pair flow is in Backbone, but almost everything else is in React and will become React in the future. 

I don't think we need to make changes over there though - looks like we're [just checking for the UA including `fxios`](https://github.com/mozilla/fxa/blob/86639ff223c5626a0b1c617918e44ead0610df24/packages/fxa-settings/src/lib/channels/firefox.ts#L220-L221)? and not stringifying it in that case.